src/Symfony/Component/VarDumper/Dumper/HtmlDumper.php

```diff
@@ -141,6 +141,11 @@ function toggle(a, recursive) {
     } else {
         return false;
     }
+    
+    s.dispatchEvent(new CustomEvent('sfbeforedumptoggle', {
```

Comment:
breaks on IE11?

Reply:
We don't care about IE for Symfony development tools (we do care about Microsoft Edge though).

Reply:
we do care about IE11 for things running in the toolbar, because this one is injected in the page itself, and so anything breaking the JS could break the page itself.

Reply:
note that today, IE11 supports the WDT properly (we just had an issue with the SVG rendering in the past doing wonky stuff when the size was not defined in the SVG, and we fixed it)

Reply:
I disagree. Symfony debug toolbars are related to the backend. If you have a Symfony-related problem in your app, you should be able to use any browser. So there's no need to support legacy browsers. The only exception are Ajax requests ... but that's two different things: you can use IE to debug frontend + JS errors related to Ajax and any other browser to debug the Symfony responses to those Ajax requests.

Reply:
At least failing gracefully should be required IMHO.

Reply:
@javiereguiluz but if the frontend is broken *because* of the JS injected by the toolbar, you cannot debug the frontend because the toolbar causes extra errors.
It is OK if some feature does not work. It is not OK for the JS to break.